internal/controller/stats.go

```diff
@@ -38,9 +38,16 @@ var (
 		Name:      "config_stale_bool",
 		Help:      "1 if running on a stale configuration, because the latest config failed to load.",
 	})
+
+	configReset = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: Namespace,
+		Subsystem: Subsystem,
+		Name:      "config_reset_total",
+		Help:      "Number of times the healthcheck controller reset the config.",
```

Comment:
Does it make sense to have this as a gauge (0 - 1) to say if the configuration was reset?

Reply:
this depends on the other discussion